content/en/blog/_posts/2024-12-11-Kubernetes-v1-32-Release.md

```diff
@@ -0,0 +1,302 @@
+
+3. The Sidecar containers feature is targeting graduation to Stable in 1.33 as the [remaining work items](https://github.com/kubernetes/enhancements/issues/753#issuecomment-2350136594) are being tracked along with feedback from users.
+
+## Highlights of features graduating to Stable
+
+_This is a selection of some of the improvements that are now stable following the v1.32 release._
+
+### Custom Resource field selectors
+
+Custom resource field selector allows developers to add field selectors to custom resources, mirroring the functionality available for built-in Kubernetes objects. This allows for more efficient and precise filtering of custom resources, promoting better API design practices.
+
+This work was done as a part of [KEP #4358](https://github.com/kubernetes/enhancements/issues/4358), by [SIG API Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery).
+
+### Support to size memory backed volumes
+
+The feature makes it possible to dynamically size memory-backed volumes based on pod resource limits. This improves the workload's portability and overall node resource utilization for the users.
+    
+This work was done as a part of [KEP #1967](https://github.com/kubernetes/enhancements/issues/1967), by [SIG Node](https://github.com/kubernetes/community/tree/master/sig-node).
+
+### Job API managed-by mechanism
+
+Support of the `managedBy` field has been added for Jobs and it is stable in 1.32 release. This feature enables external controllers (like [Kueue](https://kueue.sigs.k8s.io/)) to manage Job synchronization, offering greater flexibility and integration with advanced workload management systems.
+
+This work was done as a part of [KEP #4368](https://github.com/kubernetes/enhancements/issues/4368), by [SIG Apps](https://github.com/kubernetes/community/tree/master/sig-apps).
+
+## Features graduating to Beta
+
+_This is a selection of some of the improvements that are now beta following the v1.32 release._
+
+### Only allow anonymous auth for configured endpoints
+
+Allow users to specify which endpoints are allowed for anonymous requests. This allows the admin to only allow access to health endpoints like `/healthz`, `/livez` and `/readyz` anonymously while making sure other cluster endpoints or resources cannot be access anonymously even if a user misconfigures RBAC.
+    
+This work was done as a part of [KEP #4633](https://github.com/kubernetes/enhancements/issues/4633), by [SIG Auth](https://github.com/kubernetes/community/tree/master/sig-auth).
+
+### Per-plugin callback functions for accurate requeueing in kube-scheduler enhancements
+
+Enhance the scheduling throughput with more efficient scheduling retry decisions by per-plugin callback functions (QueueingHint). 
+We've implemented QueueingHints in all plugins, and are moving ahead enabling this feature by default.
```

Comment:
```suggestion
This feature enhance sthe scheduling throughput with more efficient scheduling retry decisions by
per-plugin callback functions (QueueingHint). All plugins now have QueueingHints.
```
Cut the bit that's true for all beta stuff

Reply:
Done!